components/molecules/navbar/components/brand/brand.js

```diff
@@ -10,22 +10,50 @@ import { string } from 'prop-types'
 import { Image, Link } from '../../../../'
 
 // Style
-import styled from 'styled-components'
+import styled, { keyframes } from 'styled-components'
```

Comment:
@zemqi Will this work on all logos we use?

Reply:
I saved standard logo and added wrapper to create logos like our DryKISS.
Also, maybe border style need to separate from component and put into drykiss.com? These places what I mean — need to refactor.

Reply:
@zemqi Remember IUI is for global users so we need to decied whether this stuff belongs in IUI or specific to us... if us then we would put in parent app..